walkthroughs/5-exploring-datasync-codeready/walkthrough.adoc

```diff
@@ -0,0 +1,434 @@
+. Your task model should be as follows
+----
+""" 
+@model
+@datasync
+"""
+type Task {
+  id: ID!
+  title: String!
+  description: String!
+  status: TaskStatus
+  address: String
+}
+----
+
+=== Generating your DataSync Node.js server and React App
+
+DataSync provides code generation capabilities that transform your model into a fully functional client and server application.
+The datasync-starter template contains the following folders:
+
+* `./server`- contains a Node.js server application written in TypeScript. `server/src/index.ts` is the entrypoint to the server application.
+* `./client` - contains a React application written in TypeScript. `client/src/index.tsx` is the entrypoint to the client application.
+* `.graphqlrc.yml` - 
+
+. Review the `.graphqlrc.yml` file. This is a config file that
+** determines what types of CRUD operations will be generated into GraphQL Queries and Mutations in your application.
+** enables and configures various plugins that are used during the code generation process.
+. Make sure that all fields in the `crud` section are enabled 
+. Execute graphback cli command to generate source code:
+`yarn graphback generate`. You can also execute it as predefined `generate source code` command in CodeReady
+. Review `./server/src/schema/schema.qraphql`. 
+This file has the original model and it also contains generated Queries and Mutations. The types of queries and mutations included are based on the `crud` fields in `.graphqlrc.yml`.
+. Review the generated resolver files in `./server/src/resolvers/resolvers.ts`
+This file contains methods used to fetch and modify data. Each individual method uses a 
+preconfigured `MongoDBDataProvider` which is an abstraction over a MongoDB client. Developers can point resolvers to any datasource.
+Currently Postgres and MongoDB are supported.
+. Review your `./client/src/graphql/` folder containing client side queries for your data. These queries are automatically generated based on the server schema, and are used within the client application. Generating the client side queries helps developers get their client apps up and running quicker and helps them stay up to date as the server schema evolves.
+
+=== Running DataSync client and server applications
+
+. Open a new terminal window
+. Execute the `prepare client` command in the new terminal. Client side application will be build and started. This can take a couple of minutes.
+. Execute the `start server` command. This command starts the GraphQL server which also serves the client application for simplicity.
+. The application should be opened in a preview window after build is finished.
+
+[type=verification]
+****
+. Check if the website was loaded properly
+. Select the + icon to create a new item
+. On the new screen enter a `name` and `description` and create the task.
+. New task should appear in the task list.
+----
```

Comment:
Is there supposed to be any code snippet? It creates this empty code snipped.
![image](https://user-images.githubusercontent.com/20477661/84382564-1ec34780-abeb-11ea-98ab-ded9100f4c50.png)

Reply:
I have noticed it. It was working in old version of the solution explorer so it might that format changed